modules/gumgumBidAdapter.md

```diff
@@ -25,6 +25,18 @@ var adUnits = [
         }
       }
     ]
+  },{
+    code: 'test-div',
+    sizes: [[0, 1], [2, 3], [4, 5], [6, 7]],
```

Comment:
This code change seems to be taking from `params.sizes` not `adUnit.sizes`

So I think the md file should have this inside the gumgum bidder params as an option.

Reply:
If not intended to be in your markdown file, maybe it should be in the prebid docs?

Not required to have it here so I will still merge.

Reply:
Correct; I removed this for now